admin-ui/src/app/views/employee/customer/customer.component.scss

```diff
@@ -0,0 +1,23 @@
+// ::ng-deep .mat-slide-toggle-bar {
```

Comment:
Remove commented lines unless necessary

Reply:
Left the CSS commented to change the size of mat-slide-toggle-bar in future